video/out/cocoa_common.m

```diff
@@ -466,14 +466,21 @@ static CVReturn displayLinkCallback(CVDisplayLinkRef displayLink, const CVTimeSt
     return kCVReturnSuccess;
 }
 
-static void vo_set_level(struct vo *vo, int ontop)
+static void vo_set_level(struct vo *vo, int ontop, int ontop_level)
 {
     struct vo_cocoa_state *s = vo->cocoa;
 
     if (ontop) {
-        // +1 is not enough as that will show the icon layer on top of the
-        // menubar when the application is not frontmost. so use +2
-        s->window_level = NSMainMenuWindowLevel + 2;
+        switch (ontop_level) {
+        case -1:
+            s->window_level = NSFloatingWindowLevel;
+            break;
+        case -2:
+            s->window_level = NSStatusWindowLevel;
+            break;
+        default:
+            s->window_level = ontop_level;
```

Comment:
on cocoa the window level can be between -INT_MAX and +INT_MAX. though everything that can be considered ontop is >0. the chart i made for the issue i mentioned.
![wlevel](https://cloud.githubusercontent.com/assets/680386/22155108/92fda664-df2e-11e6-991b-bdec97b1c693.png)